File: java/src/TwoSum.java
public class TwoSum {
    public int [] twoSum(int[] nums, int target){
        int [] pos = null;
        int i = 0;
        while (i <= (nums.length + 1)) {
            for (int k = i+1; k < nums.length; k++) {
                if (nums[i] + nums[k] == target) {
                    pos = new int[]{i, k};
                    break;
                }
            }
            if (pos == null){
                i++;
            }
            else break;
        }
        return pos;
    }

}

File: python/two-sum.py
def twoSum(numbers: list[int], target: int) -> list[int]:
    l = 0
    r = len(numbers) - 1
    while l < r:
        if numbers[l] + numbers[r] == target:
            break
        elif numbers[l] + numbers[r] > target:
            r -= 1
        else:
            l += 1
            while numbers[l - 1] == numbers[l] and l < r:
                l += 1
    return [l + 1, r + 1]


print(twoSum([2, 2, 5, 5, 7], 12))
